engine/src/main/scala/cromwell/engine/workflow/workflowstore/SqlWorkflowStore.scala

```diff
@@ -56,7 +56,7 @@ case class SqlWorkflowStore(sqlDatabase: WorkflowStoreSqlDatabase) extends Workf
       workflowStoreEntry.workflowDefinition.toRawString,
       workflowStoreEntry.workflowInputs.toRawString,
       workflowStoreEntry.workflowOptions.toRawString,
-      workflowStoreEntry.customLabels.toRawString,
+      workflowStoreEntry.customLabels.toString,
```

Comment:
Good catch. I don't think this works. Do not call `Object.toString()` on the _interface_ `Clob` or you're going to have a bad time, getting something unexpected back. `SqlConverters` needs to create a `toRawString` for `Clob`s, just like there is one for `Option[Clob]`. While there, please also add a `Blob.toBytes` implicit.